editor/plugins/shader_editor_plugin.cpp

```diff
@@ -339,7 +339,7 @@ void ShaderEditor::_menu_option(int p_option) {
 			shader_editor->remove_all_bookmarks();
 		} break;
 		case HELP_DOCS: {
-			OS::get_singleton()->shell_open("https://docs.godotengine.org/en/stable/tutorials/shading/shading_reference/index.html");
+			OS::get_singleton()->shell_open("https://docs.godotengine.org/en/stable/tutorials/shaders/shader_reference/index.html");
```

Comment:
The new link is broken, the previous was correct under the `stable` domain. The address changed in `latest` for the 4.0 doc, so if you're propagating this change here you should track `latest` (which makes sense for the `master` branch).

Reply:
Yes, I stated this in the PR description, that would be fine to merge at the 4.0 release but not before since the link is currently invalid.

Reply:
Well I think the link should point to `latest`, like for tutorial links in the class reference.

We can switch it to `stable` once we release 4.0-stable.

Reply:
Alright, changed to `latest`